### PR TITLE
INVGOV-12594 Preview mode API

### PR DESCRIPTION
**Pull request by sam-ludlow**
https://io1dev.atlassian.net/browse/INVGOV-12594

### CODE REVIEW

File: api-private.swagger.yaml [hunk 1]
```diff
@@ -1,376 +1,519 @@
-swagger: '2.0'
+openapi: 3.0.1
```
Comment (ParryLuke) [inline api-private.swagger.yaml:1]: Could we do this in a separate PR?

File: api-private.swagger.yaml [hunk 2]
```diff
@@ -1,376 +1,519 @@
...
+        201:
+          description: "Acknowledged that preview mode for the user's session has been enabled."
+          content: {}
+        400:
+          description: 'A malformed UUID was supplied.'
+          content: {}
+        404:
+          description: 'The user was not found.'
+          content: {}
+    delete:
```
Comment (ParryLuke) [inline api-private.swagger.yaml:338]: should this be a POST to include the session?

**Review by ParryLuke — CHANGES_REQUESTED** [review]

File: api-private.swagger.yaml (hunk 1 above)
Comment (sam-ludlow) [reply api-private.swagger.yaml]: reverted the upgrade

File: api-private.swagger.yaml (hunk 2 above)
Comment (sam-ludlow) [reply api-private.swagger.yaml]: made the DELETE a PUT with session body

**Review by ParryLuke — APPROVED** [review]
